release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -2647,6 +2647,38 @@ To avoid this issue, you can enable confidential VMs on an existing cluster by u
 
 * When you run hosted control planes for {product-title} on a bare-metal platform, if a worker node fails, another node is not automatically added to the hosted cluster, even when other agents are available. As a workaround, manually delete the machine that is associated with the failed worker node. (link:https://issues.redhat.com/browse/MGMT-15939[*MGMT-15939*])
 
+[id="ocp-telco-ran-4-14-known-issues"]
+* When you run CNF latency tests on an {product-title} cluster, the `oslat` test can sometimes return results greater than 20 microseconds. This results in an `oslat` test failure.
+(link:https://issues.redhat.com/browse/RHEL-9279[*RHEL-9279*])
+
+* When you use `preempt-rt` patches with the realtime kernel and you update the SMP affinity of a network interrupt, the corresponding IRQ thread doesn't immediately receive the update.
```

Comment:
Not sure if it's common knowledge, but would it help to expand on [IRQ](https://en.wikipedia.org/wiki/Interrupt_request) 
```suggestion
* When you use `preempt-rt` patches with the realtime kernel and you update the SMP affinity of a network interrupt, the corresponding interrupt request or IRQ thread doesn't immediately receive the update.
```